src/main/java/step01/StringAddCalculator.java

```diff
@@ -0,0 +1,28 @@
+package step01;
+
+import java.util.Arrays;
+
+public class StringAddCalculator {
+    private static String PATTERN = "//(.)\n(.*)";
```

Comment:
```suggestion
    private static String PATTERN = "//(.)\n(.*)";
    private static int DELIMITER = 1;
    private static int CONTENTS = 2;
```
DELIMITER랑 CONTENTS가 PATTERN의 영역에 해당하기 때문에 여기서 정의해주는게 더 자연스러울 것 같아요!